.codeclimate.yml

```diff
@@ -1,15 +1,167 @@
-engines:
-  golint:
+# This is our default .CodeClimate.yml, broken out by language. Uncomment the
+# sections at the bottom that apply to your project. ACTION comments indicate
+# places where config might need to be tweaked.
+
+version: "2"
+
+plugins:
```

Comment:
This probably needs those changes that @BradleyBoutcher applied to the broker about return statements and method lines